client/extensions/wp-super-cache/directly-cached-files.jsx

```diff
@@ -65,40 +65,40 @@ class DirectlyCachedFiles extends Component {
 					</Button>
 				</SectionHeader>
 				<Card className="wp-super-cache__directly-cached-files">
-					{ !! wp_cache_readonly &&
+					{ cache_readonly &&
 					<p>
 					{ translate(
-						'{{strong}}Warning!{{/strong}} You must make %(wp_cache_path)s wp_cache_writable to enable this feature. ' +
+						'{{strong}}Warning!{{/strong}} You must make %(cache_path)s writable to enable this feature. ' +
 						'As this is a security risk, please make it read-only after your page is generated.',
 						{
-							args: { wp_cache_path: wp_cache_path },
+							args: { cache_path: cache_path },
 							components: { strong: <strong /> },
 						}
 					) }
 					</p>
 					}
-					{ !! wp_cache_writable &&
+					{ cache_writable &&
 					<p>
 					{ translate(
-						'{{strong}}Warning!{{/strong}} %(wp_cache_path)s is wp_cache_writable. Please make it wp_cache_readonly ' +
+						'{{strong}}Warning!{{/strong}} %(cache_path)s is writable. Please make it readonly ' +
 						'after your page is generated as this is a security risk.',
 						{
-							args: { wp_cache_path: wp_cache_path },
+							args: { cache_path: cache_path },
 							components: { strong: <strong /> },
 						}
 					) }
 					</p>
 					}
 					<p>
 						{ translate(
-							'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This ' +
+							'Directly cached files are files created directly off %(cache_path)s where your blog lives. This ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Directly cached files are files created directly off %(wp_cache_path)s where your blog lives. This feature is only useful if you are expecting a major Digg or Slashdot level of traffic to one post or page.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).